src/config/client-config.c

```diff
@@ -49,94 +50,200 @@ int Read_Client(const OS_XML *xml, XML_NODE node, void *d1, __attribute__((unuse
         rip = NULL;
```

Comment:
This variable is now null. It's not necessary to reassign  `NULL`.

Reply:
This variable is assigned to `NULL` at the beginning of a loop for. It can be changed inside that loop, so it is recommendable to reassign to `NULL`.